tools/qontract_cli.py

```diff
@@ -517,6 +517,15 @@ def get_available_upgrades(ocm: OCM, version: str, channel: str) -> list[str]:
     return [u for u in upgrades if not ocm.version_blocked(u)]
 
 
+def inheritVersionDataText(ocm_org: str, ocm_specs: list[dict]) -> str:
+    ocm_spec = [o for o in ocm_specs if o["name"] == ocm_org][0]
+    inheritVersionData = ocm_spec["inheritVersionData"]
+    if not inheritVersionData:
+        return ""
+    inheritedOrgs = [f"[{o['name']}](#{o['name']})" for o in inheritVersionData]
```

Comment:
is repeating `o['name']` twice intended? the output would be something like
```
inheriting version data from [org-1](#org-1), [org-2](#org-2)
```

Reply:
yes, each org has its own section, and we're creating links to them.